src/components/BookForm.js

```diff
@@ -0,0 +1,29 @@
+import { useDispatch } from 'react-redux';
+import { addBookToApi } from '../redux/books/booksSlice';
+import BookElement from '../helpers/BookElement';
+import InputForm from './InputForm';
+
+const BookForm = () => {
+  const dispatch = useDispatch();
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const form = e.target;
+    const { inputTitle, inputAuthor } = form;
+    dispatch(addBookToApi(BookElement(inputTitle.value, inputAuthor.value)));
+    form.reset();
+  };
+
+  return (
+    <div className="book-form">
+      <h2 className="form-title">ADD NEW BOOK</h2>
+      <form onSubmit={handleSubmit} className="form">
+        <InputForm type="text" id="inputTitle" placeholder="Book title" isRequired />
+        <InputForm type="text" id="inputAuthor" placeholder="Book Author" isRequired />
+        <button type="submit" id="add-book-btn">ADD BOOK</button>
+      </form>
+    </div>
```

Comment:
Great work done by creating the InputForm component 🥇 , I have one very very small styling issues to mention : 

please make sure to use the same color for the title and the input placeholders  do your best to stick to the design 

![Capture d’écran du 2023-08-04 22-28-10](https://github.com/akbarsazish/Bookstore-React/assets/59707859/64bca33d-594e-4143-a2fa-5790cc9f37d1)


![Capture d’écran du 2023-08-04 22-27-58](https://github.com/akbarsazish/Bookstore-React/assets/59707859/13275cd6-ff6c-4a6f-9cae-35d2ef433b97)